frontend/src/static/js/v_summary.js

```diff
@@ -332,23 +333,24 @@ window.vSummary = {
       this.getDates();
       this.sortFiltered();
     },
-    processFileFormats() {
+    processFileTypes() {
       const selectedColors = ['#ffe119', '#4363d8', '#3cb44b', '#f58231', '#911eb4', '#46f0f0', '#f032e6',
           '#bcf60c', '#fabebe', '#008080', '#e6beff', '#9a6324', '#fffac8', '#800000', '#aaffc3', '#808000', '#ffd8b1',
           '#000075', '#808080'];
-      const colors = {};
+      const fileTypeColors = {};
       let i = 0;
 
       this.repos.forEach((repo) => {
         const user = repo.users[0];
-        Object.keys(user.fileFormatContribution).forEach((fileFormat) => {
-          if (!Object.prototype.hasOwnProperty.call(colors, fileFormat)) {
-            colors[fileFormat] = selectedColors[i];
+        Object.keys(user.fileTypeContribution).forEach((fileType) => {
+          if (!Object.prototype.hasOwnProperty.call(fileTypeColors, fileType)) {
```

Comment:
why didn't just `fileTypeColors[fileType]`?

Reply:
this was because `fileFormat` and `group` was previously separate, so i needed to check if object had the property first. But now that's it merged to `fileType`, i don't need to check it anymore, so i have removed the if condition.